gilmour.go

```diff
@@ -25,6 +26,11 @@ func Get(backend Backend) *Gilmour {
 	return &x
 }
 
+type RetryConf struct {
```

Comment:
@meson10 RetryConf is separate because that way it can be passed as one entity wherever needed(like in try method)

I will change RetryFrequency to RetryInterval